Makefile.am

```diff
@@ -100,6 +100,15 @@ ifp_systemdservice = SystemdService=sssd-ifp.service
 condconfigexists = ConditionPathExists=\|/etc/sssd/sssd.conf\nConditionDirectoryNotEmpty=\|/etc/sssd/conf.d/
 
 # Bounding set needs to list capabilities required by ldap/krb5/selinux_childs and sssd_pam, otherwise they can't gain it.
+# Capabilities usage by binaries:
+#   - 'ldap_child': read keytab (dac_read_search)
```

Comment:
What about updating this part of `Makefile.am` also?

~~~
CHILD_CAPABILITIES="cap_chown,cap_dac_override,cap_setuid,cap_setgid=ep"
if SSSD_USER
    -chgrp $(SSSD_USER) $(DESTDIR)$(sssdlibexecdir)/ldap_child
    chmod 750 $(DESTDIR)$(sssdlibexecdir)/ldap_child
    -$(SETCAP) $(CHILD_CAPABILITIES) $(DESTDIR)$(sssdlibexecdir)/ldap_child
~~~

Reply:
Fixed.